### PR TITLE
Solved Issue #31 Added Buttons to swap btw graphs based on Percentage and Number of Students

### PR DESCRIPTION
**Pull request by kartik1000**
Added two buttons to swap between the graph based on Percentage of Students and the one based on Number of students

### CODE REVIEW

File: templates/kronos.html [hunk 1]
```diff
@@ -148,10 +167,36 @@ <h1>Sorry, no data available for that coursecode</h1>
                     zeroline: true
                 }
             };
+            var layout1 = {
+                title: 'Grade-Distributions',
```
Comment (Ayushk4) [inline templates/kronos.html]: Can you change the title to "Grade- Distribution for {{course_code}}" or something? Also, make changes in both `layout1` and `layout`.

**Comment by Ayushk4** [comment]
I have tested this locally it works great, visually it can be made better, if you center the buttons. 
cc @xypnox

**Comment by xypnox** [comment]
Looks fine but can be improved in two ways:

- The action of these buttons is not obvious. You should have additional text such as "Show graph with respect to:" and the two buttons.

- This addition decreases the vertical space of the graph. It would be better if you could move these buttons to the side or decrease there height i some way.

**Comment by kartik1000** [comment]
Hey, rather than the side I pushed it below the graph..because the graph almost covers the entire width of the page and changed the text as suggested..Pls review

**Pull request merged**